roles/pulp_content/templates/pulpcore-content.service.j2

```diff
@@ -10,6 +10,7 @@ Environment="PATH={{ pulp_install_dir }}/bin:{{ default_bin_path }}"
 {% if pulp_ld_library_path is defined and pulp_ld_library_path != '' %}
 Environment="LD_LIBRARY_PATH={{ pulp_ld_library_path }}"
 {% endif %}
+Environment="DJANGO_ALLOW_ASYNC_UNSAFE=true"
```

Comment:
Isn't this (and the change to pulpcore-worker) a separate issue?

Reply:
If it is, and it isn't a separate commit (not necessarily a separate issue), we cannot backport it easily.